[SSL] Revise and consolidate client certs content (#23365)
cloudflare/cloudflare-docs: src/content/docs/learning-paths/mtls/mtls-app-security/index.mdx

API Shield is not required to use mTLS. <br />

> By default, mTLS uses Client Certificates issued by a Cloudflare Managed CA. Cloudflare generates a unique CA for each customer account, meaning that Client Certificates all validate against an account-level Cloudflare CA. If you have an Enterprise account, you also have the option to [bring your own CA](/ssl/client-certificates/byo-ca/).

Author: ```suggestion
By default, mTLS uses Client Certificates issued by a Cloudflare Managed CA and set at account-level. If you have an Enterprise account, you also have the option to [bring your own CA](/ssl/client-certificates/byo-ca/).
```
